handlers/dataset_test.go

```diff
@@ -89,7 +92,15 @@ func TestDatasetHandlers(t *testing.T) {
 
 				Convey("When the dataset page is rendered", func() {
 					w, req := generateRecorderAndRequest()
-					DatasetPage(mockZebedeeClient, mockRend, mockFilesAPIClient)(w, req)
+
+					cfg, err := config.Get()
+					So(err, ShouldBeNil)
+
+					ctxOther := context.Background()
```

Comment:
**COMMENT 2**
You can just use the `ctx` already defined in the test at line 43 rather creating another context (`ctxOther`)

Reply:
It doesn't like the type of ctx, if I use ctx it says:
`cannot use ctx (variable of type gomock.Matcher) as "context".Context value in argument to cache.GetMockCacheList: missing method Deadline`
Any ideas for a fix?